src/main/webapp/WEB-INF/jsp/preview.jsp

```diff
@@ -66,7 +66,7 @@
             <div class="col-sm-12">
                 <c:if test="${showConfigLink}">
                     <portlet:renderURL var="configUrl" portletMode="CONFIG"/>
-                    <a class="btn btn-primary pull-right" role="button" href="${ configUrl }"><i class="fa fa-gear" aria-hidden="true"></i> <spring:message code="preview.configure"/></a>
+                    <a class="btn btn-primary pull-right" role="link" href="${ configUrl }"><i class="fa fa-gear" aria-hidden="true"></i> <spring:message code="preview.configure"/></a>
```

Comment:
I definitly know that, but dunno if this came from CSS `transform: translate3D` double-inner off-canvas design, I was loosing the focus on a element in some old android, iPad and iPhone 4, so I have added both `role='link/button"` + tabindex on `<a>`